homeassistant/components/light/hue.py

```diff
@@ -227,7 +239,7 @@ def turn_on(self, **kwargs):
             command['alert'] = 'lselect'
         elif flash == FLASH_SHORT:
             command['alert'] = 'select'
-        else:
+        elif self.bridge_type == 'hue':
```

Comment:
The deCONZ Hue-like API doesn't handle the 'none' effect correctly.  When that attribute is present with the value none, it does random things with the lights.  With the default hub type set to 'hue' this ensures anyone but deCONZ users get to keep the default behaviour.  I could invert the condition and add an else instead:

```py
    elif self.bridge_type == 'deconz':
       pass
    else:
       command['effect'] = 'none'
```